lua/telescope/builtin/git.lua

```diff
@@ -111,9 +111,7 @@ git.status = function(opts)
   for _, v in ipairs(output) do
     if v ~= "" then
       local mod, fname = string.match(v, '(..)%s(.+)')
-      if mod ~= 'A ' and mod ~= 'M ' and mod ~= 'R ' and mod ~= 'D ' then
         table.insert(results, { mod = mod, file = fname })
```

Comment:
indentation should be wrong now

Reply:
For me Indentation still works, as the space is significant: without colors, it's currently the only way to distinguish between staged and unstaged files

Reply:
i meant the line indentation from table.insert. Two spaces too much. Sorry about nitpicking

Reply:
Don't be sorry, I'm here for that kind of precision! I have to fix my indentation for lua files, but when you spot blunders like that, I'm thankful more than anything ^^